### DISCUSSION TITLE
How to use CGAL 5.6.1?

### DISCUSSION THREAD

**Discussion by Eric5709** [CGAL/cgal#8083] [Q&A] [answered]
Hello, I'm newbie to C++ and CMake.

I'm facing an issue where I wnat to use CGAL version 5.6.1, but my Ubuntu 20.04 environment comes with the default version of CGAL, which is 5.0.2.
I've downloaded the 5.6.1 tar.xz and set the directory in the CMakelist.txt file but the version of CGAL is still 5.0.2.
Here is the CMakeLists.txt.(I'm using CLion)

```
cmake_minimum_required(VERSION 3.8)
project(test)

if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
  add_compile_options(-Wall -Wextra -Wpedantic)
endif()

...

SET(CGAL_DIR "/home/jsh/.sh/cgal/CGAL_5.6.1")           // where "CGALConfig.cmake" is located.
find_package(CGAL REQUIRED)
message(STATUS "CGAL version: ${CGAL_VERSION}")
```

Is it possible to use CGAL version 5.6.1 like this? If not, please tell me how to do so.

**Comment by sloriot** [comment, marked as answer]
It is pretty surprising to extract the CGAL archive in the .sh directory.

The simplest solution to use a CGAL release freshly extracted is:
```
PWD=`pwd`
wget https://github.com/CGAL/cgal/releases/download/v5.6.1/CGAL-5.6.1.tar.xz
tar xf CGAL-5.6.1.tar.xz
mkdir build_t2_ex
cd build_t2_ex
CGAL_DIR=$PWD/CGAL-5.6.1 cmake $PWD/CGAL-5.6.1/examples/Triangulation_2
make
```
